scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-1a.yaml

```diff
@@ -106,7 +106,6 @@ proposals:
       backend_name: nfs
       nfs:
         nfs_shares: ##cinder-storage-shares##
-        nfs_mount_options: nfsvers=3,rw,sync,nofail
```

Comment:
If you want to have those options in your setup, you need to add them into the `nfs_shares` value somehow.
Now I'm not sure how your `##cinder-storage-shares##￼` looks like, but that's the value that could be extended